components/post_view/reaction_list/index.js

```diff
@@ -34,8 +37,9 @@ function mapDispatchToProps(dispatch) {
     return {
         actions: bindActionCreators({
             addReaction,
+            removeReaction,
         }, dispatch),
     };
 }
 
-export default connect(makeMapStateToProps, mapDispatchToProps)(ReactionList);
+export default connect(makeMapStateToProps(), mapDispatchToProps)(ReactionList);
```

Comment:
`makeMapStateToProps` shouldn't be called by us or else you end up in the same situation as just having a single `mapStateToProps` function where selectors are overwriting each others' results.